cs/costcalculator.go

```diff
@@ -204,13 +204,16 @@ func (p *costCalculate) CostOfOne(player *Player, item ProductionQueueItem) (Cos
 // Get cost of a given ship or new starbase design
 func (p *costCalculate) GetDesignCost(rules *Rules, techLevels TechLevel, raceSpec RaceSpec, design *ShipDesign) (Cost, error) {
 
+	starbase := false
 	hull := rules.techs.GetHull(design.Hull)
 	if hull == nil {
 		return Cost{}, fmt.Errorf("hull design %s not found in tech store", design.Hull)
+	} else if hull.Starbase {
```

Comment:
This shouldn’t be an else if. It’s not related to the nil check. Just make it an if.